src/global.d.ts

```diff
@@ -0,0 +1,37 @@
+import * as PIXI from 'pixi.js';
+import GestureEvent from './gesture-event';
+
+declare global {
+  interface GestureEventMap extends HTMLElementEventMap {
+    gesturestart: GestureEvent;
+    gesturechange: GestureEvent;
+    gestureend: GestureEvent;
+  }
+
+  interface HTMLElement {
+    addEventListener<K extends keyof GestureEventMap>(
```

Comment:
Not sure this is quite strictly right- won’t this stop listeners for non-gesture stuff ? Not a big deal in this context but worth considering

Reply:
`GestureEventMap extends HTMLElementEventMap` for this very reason! 👍

Reply:
Aha, missed that part